java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -3250,6 +3250,19 @@ private static void assertIsSupportedMapKeyType(DType keyType) {
     assert isSupportedKeyType : "Map lookup by STRUCT and LIST keys is not supported.";
   }
 
+  /**
+   * Given a column of type List<Struct<X, Y>> and a key of type X, return a column of type Y,
+   * where each row in the output column is the Y value corresponding to the X key.
+   * If the key is not found, the corresponding output value is null.
+   * @param keys the column view with keys to lookup in the column
+   * @return a column of values or nulls based on the lookup result
+   */
+  public final ColumnVector getMapValueForKeys(ColumnView keys) {
```

Comment:
Why not just call it `getMapValue` just like the Scalar API is called? If they do the same thing, then to me it is ideal to call them the same thing.